lib/models/theme.dart

```diff
@@ -16,6 +16,31 @@ class DialogOption<T> {
   DialogOption({this.value, this.widget});
 }
 
+class SupportedLocales {
+  static const en = 'en';
+  static const hi = 'hi';
+  static const es = 'es';
+  static const nb_NO = 'nb_NO';
+  static const pt_BR = 'pt_BR';
+  static const zh_Hans = 'zh_Hans';
+  static const values = [
+    SupportedLocales.en,
+    SupportedLocales.hi,
+    SupportedLocales.es,
+    SupportedLocales.nb_NO,
+    SupportedLocales.pt_BR,
+    SupportedLocales.zh_Hans,
+  ];
+  static const Map<String, String> languageNameExpanded = {
+    'en': 'English',
+    'hi': 'Hindi',
+    'es': 'Spanish',
+    'nb_NO': 'Norwegian Bokmål (Norway)',
+    'pt_BR': 'Portuguese (Brazil)',
+    'zh_Hans': 'Chinese (Simplified Han)'
```

Comment:
Should the corresponding text for each language be written in that language?

For example, Spanish -> Español

Reply:
Perhaps Google translate could help